pkg/runctl/run_manager.go

```diff
@@ -42,6 +45,11 @@ const (
 	// tektonTaskRun is the name of the Tekton TaskRun in each
 	// run namespace.
 	tektonTaskRunName = "steward-jenkinsfile-runner"
+
+	// repeatGetServiceAccountSecretName can be set to false to debug hanging tests
+	// if set to false getServiceAccountSecretName will panic if no stub is defined and no service account secret can be found
+	// for productive usage set to true, this will allow a retry to get the service account secret as soon as it is available
+	repeatGetServiceAccountSecretName = true
```

Comment:
Ugly. There's the risk that after a successful test with `false` this code gets committed, breaking functionality in production.

Reply:
Removed